packages/e2e-test-utils/src/publishStory.js

```diff
@@ -21,19 +21,19 @@
  * @return {Promise<void>}
  */
 async function publishStory(dismiss = true) {
-  await expect(page).toClick('button', { text: 'Publish' });
+  await expect(page).toClick('button', { text: /^Publish$/ });
```

Comment:
New 'publishing' panel was added, so this was matching the button in that panel.